main.tf

```diff
@@ -69,11 +69,11 @@ resource "google_project_iam_member" "for_lacework_service_account" {
 
 // Roles for an ORGANIZATION level integration
 resource "google_organization_iam_custom_role" "lacework_custom_organization_role" {
-  role_id     = "lwOrgComplianceRole"
+  role_id     = "lwOrgComplianceRole_${random_id.uniq.hex}"
   org_id      = var.organization_id
   title       = "Lacework Org Compliance Role"
   description = "Lacework Org Compliance Role"
-  permissions = ["bigquery.datasets.get", "pubsub.topics.get", "storage.buckets.get"]
+  permissions = ["bigquery.datasets.get", "compute.projects.get", "pubsub.topics.get", "storage.buckets.get"]
```

Comment:
It feels odd to add permissions in this PR, I am not saying we shouldn't because you both are the experts in GCP 😉but I just want to understand why do we need them so that when users notice new roles are being assigned, we can explain why...